lib/trento_web/openapi/v1/schema/user.ex

```diff
@@ -4,6 +4,61 @@ defmodule TrentoWeb.OpenApi.V1.Schema.User do
   require OpenApiSpex
   alias OpenApiSpex.Schema
 
+  defmodule UserProfile do
+    @moduledoc false
+
+    @schema %Schema{
+      title: "UserProfile",
+      description: "Trento User profile",
+      type: :object,
+      additionalProperties: false,
+      properties: %{
```

Comment:
Shouldn't we have the `abilities` here?
Or do we include them in a next PR?
Yes, we can do later on, once the `abilities` schemas and code are ready. Related to: https://github.com/trento-project/web/pull/2568

Reply:
Let's include when the abilities pr code is ready